Hints.py

```diff
@@ -876,6 +916,15 @@ def buildBingoHintList(boardURL):
     return hints
 
 
+def alwaysNamedItem(world, locations):
```

Comment:
I'll need help for what to put as a description because other than the part for bingo, I have no idea how to formulate what this function does (that function was basically a part of the code that was turned into a method to avoid repetition)